modules/ROOT/pages/deploy-partner-manager.adoc

```diff
@@ -0,0 +1,39 @@
+= Deploying Partner Manager
+
+You can deploy Anypoint Partner Manager to either CloudHub or Anypoint Runtime Fabric. 
+
+To configure Partner Manager deployment settings, use the *Partner Manager Settings* page in Access Management. You can configure separate deployment settings for each business group environment. You must have the `Partner Manager Administrator` permission to configure or modify deployment settings.
+
+== Deploy Partner Manager to a New Environment 
+
+You must configure Partner Manager settings for a new environment if there is no Partner Manager configuration for any environment in a particular business group:
+
+include::partial$access-mgmt-steps-partial.adoc[]
+
+. Click *Partner Manager* and then click *Configure new environment*.
+. Select the environment and deployment target types.
+. In the *Configure a Partner Manager environment* window, configure either xref:cloudhub-deployment-settings.adoc[CloudHub deployment settings] or xref:rtf-deployment-settings.adoc[Runtime Fabric deployment settings].
+
+== Deploy Partner Manager to an Existing Environment
+
+Follow these steps to deploy Partner Manager to an existing business group environment:
+
+include::partial$access-mgmt-steps-partial.adoc[]
+
+. Click *Partner Manager*.
+. In the *Configure a Partner Manager environment* window, configure seither xref:cloudhub-deployment-settings.adoc[CloudHub deployment settings] or xref:rtf-deployment-settings.adoc[Runtime Fabric deployment settings].
+
+== Modify Partner Manager Deployment Settings
+
+You can modify all Partner Manager deployment settings for an environment if you haven't deployed any message flows in that environment.
```

Comment:
Runtime version and Application capacity settings (vCore for CloudHub, CPU/Memory settings for RTF) can be changed even when there are deployed flows in the environment.
Below are the configurations that cannot be changed when there are deployed message flows:
- Deployment target (from CH to RTF, or vice versa)
- If Deployment target is CloudHub: CH Deployment Region cannot be changed when there are actively deployed message flows
- If Deployment target is RTF: Application name prefix & Anypoint MQ region cannot be changed when there are actively deployed message flows